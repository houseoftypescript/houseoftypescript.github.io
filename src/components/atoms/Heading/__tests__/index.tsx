import { render } from '@testing-library/react';
import { Heading } from '../index';

describe('Heading', () => {
  it('renders correctly', () => {
    const button = render(<Heading></Heading>);
    expect(button.container).toMatchSnapshot();
  });
});
