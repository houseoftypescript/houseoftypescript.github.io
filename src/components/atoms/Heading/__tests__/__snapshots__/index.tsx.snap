// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading renders correctly 1`] = `
<div>
  <div
    class="mb-8"
  >
    <div
      class="text-center max-w-lg mx-auto"
    >
      <h1
        class="text-lg sm:text-2xl mb-4 font-b162 font-bold"
      />
      <h2
        class="text-sm sm:text-lg"
      />
    </div>
  </div>
</div>
`;
